modules/extension/xsd/xsd-wcs/src/main/java/org/geotools/wcs/v1_1/WCS.java

```diff
@@ -0,0 +1,221 @@
+/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
```

Comment:
You will need to change to the GeoTools header. The change from GPL --> LGPL is normally fraught with peril (i.e. rewriting) but since we have code contributions in place a GeoServer PSC member can give you permission to port the code and relicense under LGPL.

So send an email to geoserver-devel asking to back port to GeoTools and I can give you a +1 in public.